README.md

```diff
@@ -13,5 +218,65 @@ Currently, we support the following protocols:
 
 * [WebSocket System Handle](https://github.com/eProsima/WebSocket-SH)
 
-For more information, please refer to the [official documentation](https://integration-service.docs.eprosima.com/en/latest/).
+Additionally, creating a *System Handle* is a relatively easy task and allows to integrate a new
+protocol to the *Integration System* infrastructure, meaning that this new protocol will automatically
+be granted with communication capabilities with all of the middlewares and protocols aforementioned above.
+
+For more information, please refer to the [System Handle creation tutorial](<!-- TODO: ADD LINK TO SH CREATION TUTORIAL -->) available in the official documentation.
+
+# Compilation flags
+
+The *Integration Service* uses `CMake` for building and packaging the project.
+There are several CMake flags, which can be tuned during the configuration step:
+
+* `BUILD_TESTS`: When compiling the *Integration Service*, use the `-DBUILD_TESTS=ON` CMake option
+  to compile both the unitary tests for the [Integration Service Core](core/) and the unitary
+  and integration tests for all of the *System Handles* present in the *colcon* workspace:
+
+  ```bash
+  ~/is_ws$ colcon build --cmake-args -DBUILD_TESTS=ON
+  ```
+
+* `BUILD_EXAMPLES`: Allows to compile utilities that can be used for the several provided
+  usage examples for the *Integration Service*, located under the [examples/utils](examples/utils/) folder.
+  This applications can be used to test the *Integration Service* with some of the provided YAML configuration
+  files, which are located under the [examples/basic](examples/basic) directory:
+
+  ```bash
+  ~/is_ws$ colcon build --cmake-args -DBUILD_EXAMPLES=ON
+  ```
+  <details>
+  <summary>Up to this time, the following user application examples are available: <i>(click to expand)</i></summary>
+
+  * `DDSHelloWorld`: A simple publisher/subscriber application, running under [Fast DDS](https://fast-dds.docs.eprosima.com/).
+    It publishes or subscribes to a simple string topic, named *HelloWorldTopic*. To compile it,
+    besides using colcon, the following commands can be executed:
```

Comment:
```suggestion
    besides using colcon, the following commands need to be executed:
```

Reply:
There is no need if you already used colcon using the -DBUILD_EXAMPLES flag because it should automatically compile the example for you. This is an alternative to colcon, I am going to rephrase a little bit

Reply:
Ok. Something like. "As an alternative to colcon, in order to compile the `DDSHelloWorld` example, you can execute the following commands:"